config/packages/cache.yaml

```diff
@@ -16,3 +16,20 @@ framework:
                 adapter: cache.adapter.redis
                 # Default expire set to 5 minutes
                 default_lifetime: 300
+
+            # Creates a "auth-screen.cache" service
+            authscreen.cache:
```

Comment:
This can be autowired as `$authscreenCache` (https://symfony.com/doc/current/cache.html#creating-custom-namespaced-pools)

Reply:
fixed